repos/katello-repos.spec

```diff
@@ -46,7 +46,8 @@ install -m 644 %{SOURCE1} %{buildroot}%{_sysconfdir}/yum.repos.d/
 install -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-katello
 
 for repofile in %{buildroot}%{_sysconfdir}/yum.repos.d/*.repo; do
-    sed -i 's/@DIST@/%{dist}/' $repofile
+    trimmed_dist=`echo %{dist} | sed 's/.//'`
```

Comment:
This will trim the first character always. 's/\.//' would just strip '.'. Not sure if that matters here or not.